src/server/api/routers/user.ts

```diff
@@ -0,0 +1,73 @@
+import { z } from 'zod';
+
+import { createTRPCRouter, publicProcedure } from '~/server/api/trpc';
+
+export const userRouter = createTRPCRouter({
+  getAllUsers: publicProcedure.query(async ({ ctx }) => {
+    const users = await ctx.prisma.user.findMany();
+
+    return users;
+  }),
+
+  createUser: publicProcedure
+    .input(
+      z.object({
+        name: z.string(),
+        email: z.string(),
+        password: z.string(),
+      }),
+    )
+    .mutation(async ({ input, ctx }) => {
+      const newUser = await ctx.prisma.user.create({
+        data: {
+          name: input.name,
+          email: input.email,
+          password: input.password,
+        },
+      });
+
+      return newUser;
+    }),
+
+  getUser: publicProcedure
+    .input(z.object({ email: z.string(), password: z.string() }))
+    .query(async ({ input, ctx }) => {
+      const user = await ctx.prisma.user.findUnique({
+        where: { email: input.email, password: input.password },
+      });
+
+      return user;
+    }),
```

Comment:
I would call this login. And just so you know, this is definitely just the first iteration of login.